### ISSUE TITLE
The null object does not have a getter 'parameters'

### ISSUE THREAD

**Issue by wienczny** [dart-lang/reflectable#24]
When running the latest reflectable transformer on my project I got this exception:
```
Build error:
Transform Reflectable on test|ReflectableTransformed threw error: The null object does not have a getter 'parameters'.

NoSuchMethodError: method not found: 'parameters'
Receiver: null
Arguments: []
dart:core-patch/object_patch.dart 42                             Object._noSuchMethod
dart:core-patch/object_patch.dart 45                             Object.noSuchMethod
package:reflectable/src/transformer_implementation.dart 1776:39  TransformerImplementation._generateNewEntryPoint
package:reflectable/src/transformer_implementation.dart 1877:11  TransformerImplementation.apply.<async>
dart:async/zone.dart 1166                                        _RootZone.runUnary
dart:async/future_impl.dart 494                                  _Future._propagateToListeners.handleValueCallback
dart:async/future_impl.dart 577                                  _Future._propagateToListeners
dart:async/future_impl.dart 368                                  _Future._completeWithValue
dart:async/future_impl.dart 422                                  _Future._asyncComplete.<fn>
dart:async/schedule_microtask.dart 43                            _microtaskLoop
dart:async/schedule_microtask.dart 52                            _microtaskLoopEntry
dart:isolate-patch/isolate_patch.dart 96                         _runPendingImmediateCallback
dart:isolate-patch/isolate_patch.dart 149                        _RawReceivePortImpl._handleMessage

dart:core                                                        Object.noSuchMethod
package:reflectable/src/transformer_implementation.dart 1776:39  TransformerImplementation._generateNewEntryPoint
package:reflectable/src/transformer_implementation.dart 1877:11  TransformerImplementation.apply.<async>
```
I'm using the latest git commit #f130cb6486f221e7a1907d9d3e5a642095b1e671 and Dart VM version: 1.12.0 (Mon Aug 31 03:24:52 2015) on "linux_x64"

**Comment by sigurdm** [comment]
It looks like the transformer is run on an entry-point without a main function.
It should of course not crash, but just give an error message.

How are you using the transformer?
Does the entry-point file specified in your pubspec.yaml have (or export) a main function?

**Comment by wienczny** [comment]
I was trying to find the reason for #25 and had a lot of commented code lines. So it could be that main was missing but I'm not sure about that. This exception just occoured once.
I'm was using configuration similar to the one from https://github.com/dart-lang/sample-todomvc-polymer/tree/polymer-0.17-behaviors

**Comment by sigurdm** [comment]
Fixed in 8bc54da05cd516de802766eacbef85c7e2adcda6

**Comment by sigurdm** [comment]
I believe that must have been the case.
Thanks for reporting!
